action.yml

```diff
@@ -8,7 +8,7 @@ inputs:
   schemaPath:
     description: "The path to the plugin settings schema."
     required: false
-    default: "${{ github.workspace }}/src/types/plugin-input.js"
+    default: "${{ github.workspace }}/src/types/plugin-input.ts"
```

Comment:
Good that you're executing ts directly

Reply:
I used to have to get the JS because of `tsc`. Now I just use the `ncc` compilation result.